grid/project-management/tasks/profile.supr

```diff
@@ -0,0 +1,71 @@
+"""
+Manage tasks in project management systems.
+"""
+name = "project-management/tasks"
+version = "1.0.0"
+
+"""
+Create a new task.
+"""
+usecase CreateTask {
+	input {
+
+    """
+    projects
+    An array of IDs of projects this task belongs to.
+    """
+    projects! [string]!
+
+		"""
+		label
+		Textual label of the task.
+		"""
+		label! string!
+		
+		"""
+		description
+		Free-form textual description of the task.
+		"""
+		description string
+	
+    """
+    parent
+    ID of the parent task (if applicable).
+    """
+    parent string
+
+    """
+    assignee
+    ID of the user this task should be assigned to.
+    """
+    assignee string
+	}
+
+	result {
+    "ID of the just created task."
+    id! string!
+
+    "Project IDs it was successfully added to."
+    projects! [string]!
```

Comment:
This seems redundant?

Reply:
Not if the task failed to be added in some, but not other projects. This isn't the case for Asana which simply fails the whole operation, but it may be the case for other providers the maps of which would be creating multiple issues as described above. (If that's removed though, then yes, this is redundant.)